# Robot initial positions and configurations
robot_positions:
  - id: 1
    x: 0.0
    y: 0.0
    z: 0.01
    yaw: 0.0
  - id: 2
    x: 2.0
    y: 0.0
    z: 0.01
    yaw: 0.0
  - id: 3
    x: 4.0
    y: 0.0
    z: 0.01
    yaw: 0.0 